src/modules/match-attendances/match-attendances.controller.ts

```diff
@@ -80,15 +81,13 @@ export class MatchAttendancesController {
     return formatSuccessResponse(message, attendance);
   }
 
-  @Patch(':matchId')
+  @Patch()
```

Comment:
We need to name this endpoint somehow so that it is obvious that it's purpose is to leave the currently active match. As it is right now, it's just a PATCH  `/match-attendances` endpoint with no parameters, there's no way someone from outside will know how to use it. In fact, we will forget in a week 🤣

Reply:
yeah at first I got lost looking into this😅

Reply:
I feel like the changes you suggested in the docs are not enough. I mean, we can have descriptions in swagger, there's no harm in that, but I think the endpoint path should be descriptive as well. 

```suggestion
  @Patch('leave-active')
```